assets/src/blocks/Covers/Covers.js

```diff
@@ -1,11 +1,8 @@
-import {ContentCovers} from './ContentCovers';
 import {TakeActionCovers} from './TakeActionCovers';
 import {COVERS_TYPES} from './CoversConstants';
 
 export const Covers = ({cover_type, ...props}) => {
   switch (cover_type) {
-  case COVERS_TYPES.content:
```

Comment:
It should now always be Take Action type, right? Can't we remove this whole `switch` block?

Reply:
[Fixed!](https://github.com/greenpeace/planet4-master-theme/pull/2461/commits/53320971cf54c3e706bd838c69154118ca65be2d) Thanks